src/driver/kvm.rs

```diff
@@ -155,6 +155,14 @@ impl<T: KVMIntrospectable> Introspectable for Kvm<T> {
             ss: sregs.ss.into(),
             tr: sregs.tr.into(),
             ldt: sregs.ldt.into(),
+            idt: SystemTableReg {
```

Comment:
nitpick: It would be great if you could implement the `From` trait here similar to the segment regs.

Reply:
Good idea, thanks !